OpenSim/Examples/OptimizationExample_Arm26/OptimizationExample.cpp

```diff
@@ -70,12 +69,13 @@ class ExampleOptimizationSystem : public OptimizerSystem {
         osimModel.updDefaultControls() = newControls;
                 
         // Integrate from initial time to final time
-        p_manager->setInitialTime(initialTime);
-        p_manager->setFinalTime(finalTime);
+        Manager manager(osimModel, *p_integrator);
```

Comment:
I think it was more specifically reuse of the `TimeStepper`. Before, a `TimeStepper` was created and initialized each time in the `Manager::integrate(s)` function. But now, you pass the initial state into the `Manager`, and now the `TimeStepper` isn't aware of it (since a new one isn't created).